guides/common/modules/con_content-flow-in-projectname.adoc

```diff
@@ -0,0 +1,28 @@
+[id="Content-Flow-in-{ProjectNameID}_{context}"]
+= Content Flow in {ProjectName}
+
+Content flow in {ProjectName} involves management and distribution of content from external sources to the managed systems.
+
+Content in {Project} flows from _external content sources_ to _{ProjectServer}_.
+_{SmartProxyServers}_ mirror the content to _managed hosts_.
+
+*External content sources*::
+You can configure many content sources with {Project}.
+The supported content sources include the Red{nbsp}Hat Customer Portal, Git repositories, Ansible collections, Docker Hub, Puppet Forge, SCAP repositories, or internal data stores of your organization.
+*{ProjectServer}*::
+On the {ProjectServer}, you plan and manage the content life cycle.
```

Comment:
I don't think that "lifecycle" as one word is correct. According to the Merriam-Webster dictionary, which is recommended by the IBM SG, it should be two words.

Reply:
@Lennonka Since the UI spells it out as one word, I ended up replacing it throughout documentation because it was inconsistently spelled "life cycle" and "lifecycle".  Refer to this PR: https://github.com/theforeman/foreman-documentation/pull/2633